webrtc/videoroom/lib/videoroom_web/room_channel.ex

```diff
@@ -85,11 +85,30 @@ defmodule VideoRoomWeb.RoomChannel do
     {:noreply, socket}
   end
 
+  @impl true
+  def handle_info({:signal, {:replace_track, old_track_id, new_track_id}}, socket) do
+    push(socket, "replaceTrack", %{
+      data: %{
+        "type" => "replaceTrack",
```

Comment:
do we need this "type" if we don't support any other type in `replaceTrack` event?